salt/states/netconfig.py

```diff
@@ -19,16 +18,18 @@
 """
 
 # Import Salt libs
-from __future__ import absolute_import, print_function, unicode_literals
 
-import logging
+try:
+    import logging
```

Comment:
Why would the logging module not be available?

Reply:
I have no idea, thank you for bringing this up as I forgot. The salt-lint (I think it was that one?) pre-commit check flagged this as an external module that wasn't wrapped in a try/except... but the `logging` module is 100% internal afaik. I'll totally fix if you show me how :)

Reply:
Hmm, leave it as it was and let's see if there are still complains on the CI pre-commit